pixeltree/MyFirstBlogApi
# Added delete endpoint

# Description 

Adds an endpoint to delete a single post. 

## Testing

1. Start server using your ide or dotnet run from within the `MyFirstBlogApi/MyFirstBlog` directory.
2. Navigate to `https://localhost:5001/swagger/index.html`
3. Create a post using the swagger interface
4. Delete the post using the posts slug
5. Ensure post is properly deleted
